pages/blog/[id].js

```diff
@@ -0,0 +1,80 @@
+import { getAllPostIds, getPostData } from '../../lib/posts';
+import Head from 'next/head';
+import Date from '../../components/date'
+
+export default function Post({ post }) {
+  return (
+    <div className="mx-auto px-4 sm:px-6 lg:px-8 padding-bottom-10">
+      <Head>
+        <title>{post.title}</title>
+        <meta
+          name="description"
+          content={post.description}
+          key="description"
+        />
+
+        <meta 
+          property="twitter:title"
+          content={post.title}
+          key="tw_title"/>
+        <meta 
+          property="twitter:description"
+          content={post.description}
+          key="tw_description" />
+        
+        <meta
+          property="og:title"
+          content={post.title}
+          key="og_title"/>
+        <meta
+          property="og:description"
+          content={post.description}
+          key="og_description" />
+      </Head>
+      <div className="py-16 bg-white overflow-hidden">
+      <div className="px-4 sm:px-6 lg:px-8">
+        <div className="text-lg max-w-prose mx-auto mb-6">
+          <h1 className="mt-2 mb-8 text-3xl text-center leading-8 font-extrabold tracking-tight text-gray-900 sm:text-4xl sm:leading-10">
```

Comment:
I'd expect an ID on the most important tags like this for customisation via CSS.

Reply:
@welteki thoughts?